apps/docs/src/pages/components/container.mdx

```diff
@@ -0,0 +1,87 @@
+import {
+  containerClasses,
+  HvContainer,
+  HvTypography,
+} from "@hitachivantara/uikit-react-core";
+
+import Playground from "@docs/components/code/Playground";
+import { Description } from "@docs/components/page/Description";
+import { Page } from "@docs/components/page/Page";
+import { getComponentData } from "@docs/utils/component";
+
+export const getStaticProps = async ({ params }) => {
+  const meta = await getComponentData("Container", "core", containerClasses);
+  return { props: { ssg: { meta } } };
+};
+
+<Description />
+
+<Page>
+
+```tsx live
+import { useState } from "react";
+import { useWidth } from "@hitachivantara/uikit-react-core";
+
+export default function Demo() {
+  const [maxWidth, setMaxWidth] = useState("md");
+  const [fixed, setFixed] = useState(false);
+  const [disableGutters, setDisableGutters] = useState(false);
+  const width = useWidth();
```

Comment:
I'm not a fan of this. 60 lines just to show the current width looks excessive.
I think we should leverage our `Playground` for this.
[MUI also doesn't render any content inside the container](https://mui.com/material-ui/react-container/)

Reply:
I agree, I've redone it.